apps/condo/domains/acquiring/schema/RecurrentPaymentContext.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Generated by `createschema acquiring.RecurrentPaymentContext 'limit:Text;paymentDay:Integer;settings:Json;billingCategories:Json'`
+ */
+const { Text, Relationship, Integer, Checkbox } = require('@keystonejs/fields')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/RecurrentPaymentContext')
+const { RECURRENT_PAYMENT_CONTEXT_BOTH_TRIGGER_SET_UP_ERROR } = require('@condo/domains/acquiring/constants/errors')
+
+
+const RecurrentPaymentContext = new GQLListSchema('RecurrentPaymentContext', {
+    schemaDoc: 'Recurrent payments context configuration. Holding settings, limits and etc, configured by resident for background process that proceeding resident`s receipts on monthly basis.',
+    fields: {
+        enabled: {
+            schemaDoc: 'Enable or Disable recurrent payment proceeding.',
+            type: Checkbox,
+            defaultValue: true,
+        },
+
+        limit: {
+            schemaDoc: 'Maximal `toPay` amount of multi payment that can be proceeded. Otherwise payment should be interrupted.',
+            type: Text,
+            isRequired: false,
+        },
+
+        autoPayReceipts: {
+            schemaDoc: 'Pay for billing receipts right after they was created. Only one trigger should be configured: autoPayReceipts or paymentDay.',
+            type: Checkbox,
+            defaultValue: false,
+        },
+
+        paymentDay: {
+            schemaDoc: 'The day of month when resident`s receipts going to be proceeded. Only one trigger should be configured: autoPayReceipts or paymentDay.',
+            type: Integer,
+            isRequired: false,
+            hooks: {
+                validateInput: ({ resolvedData, addFieldValidationError }) => {
+                    if (resolvedData['autoPayReceipts'] && resolvedData['paymentDay']) {
+                        addFieldValidationError(RECURRENT_PAYMENT_CONTEXT_BOTH_TRIGGER_SET_UP_ERROR)
+                    }
+                },
+            },
+        },
+
+        settings: {
+            schemaDoc: 'Hold additional settings for recurrent payment background process.',
```

Comment:
If this is not typed its best to specify a reason for that in schemaDoc. Not typed because of 3rd party integrations